pkg/sharding/ranges_test.go

```diff
@@ -388,3 +396,234 @@ func TestLogRanges_AllShards(t *testing.T) {
 		})
 	}
 }
+
+func Test_logRangesFromPath(t *testing.T) {
```

Comment:
Can you please collapse the above tests `TestNewLogRanges`, `TestLogRangesFromPathJSON`, and `TestLogRangesFromPath` into this table structure? Otherwise it is not clear to a reader why we have different tests with similar names and testing similar things.

Reply:
Took care of it.